src/Infrastructure/Repository/BilingualRepository.php

```diff
@@ -125,13 +125,16 @@ final protected function bilingualOption(string $key): MultilingualString
         );
     }
 
-    final protected function bilingualLinkFromTerm(?WP_Term $term): MultilingualLink
+    final protected function bilingualLinkFromTerm(?WP_Term $term, string $taxonomy): MultilingualLink
     {
+        $parentTerm = $term instanceof WP_Term && $term->parent ? get_term($term->parent, $taxonomy) : null;
```

Comment:
Is it really required to have the `$taxonomy` here? Given that the `$term` argument and the possible `$parent` belong to the same taxonomy, I believe you can use `$term->taxonomy` instead, when retrieving the parent term.

If this can be dropped indeed, then the added taxonomy keys in the `WordPressDatabaseDegreeProgramRepository` are redundant, and should be removed as well.